### ISSUE TITLE
About & FAQ Pages (1 SP)

### ISSUE THREAD

**Issue by mattsayre** [Hack4Eugene/SpeedUpAmerica#233]
Add traditional top of page primary navigation, replacing buttons for FB, Twitter, Linkedin with links to sub pages. Create and add links to two new sub pages  'About" and 'FAQ.  Add additional links to pages “View Map” and “Take the Test”. Example nav inplementation at speedupsanjose.com

Content for these two pages has been drafted and can be found here: https://drive.google.com/drive/u/2/folders/1IBkbK8LU7A4NYfiNKhgssaklLBjF0Z_S

**Comment by noahbrenner** [comment]
A couple questions for you, @mattsayre:

1. Do we have a mockup, or do you have an idea in mind, for how to structure & place the nav links? You mentioned *replacing* the social media buttons. The San Jose site still has those buttons but has nav links in the very top bar (which close into a hamburger menu on narrow viewports like phones).

2. I see the Google Drive document for the FAQ, but I'm not sure which one has the content for the About page. Is the first question of the FAQ meant to be that and on it's own page?

This does look a little like 2 separate tickets (nav and new pages), but I'm happy to tackle them together.

**Comment by mattsayre** [comment]
The issue that we're working to resolve is to make it more obvious that "Take the Test" and "View Results" are the two things that we want the user to hone in on.  Based on that primary purpose, the social buttons are a distraction. The word replace was not the best choice on my part. Better to just say remove those social buttons.

Yes on the About page. Parse that "How did SpeedUpAmerica come to be?" section as the content for that page. Then the rest of the document is the content for the FAQ.

**Comment by alexandracorvello** [comment]
Hi Noah,
Agree with Matt and apologies about the google doc confusion. I don't know if this would be a separate issue but if you are working on the home page could you get rid of the "Questions about speedupamerica.com? Find more information here." (see attached file or at least I hope I attached a file). That link to GitHub has been included in the FAQs page.  Thank you!

**Comment by noahbrenner** [comment]
Thanks to both of you for the clarifications! I haven't officially assigned myself this ticket yet, but I plan to do so when I have a little time, likely within the next week or so (unless someone else has time sooner and takes the ticket).

@alexandracorvello -- I'm not seeing an attached file. Would you try either providing a link to it or writing your comment directly on the GitHub website, rather than by email, and dragging the file into the comment box?

**Comment by noahbrenner** [comment]
Looking over this ticket again, I'm thinking the scope is creeping a bit and we *would* do well to separate it into several issues. That will also help compartmentalize our discussions about each aspect. Unless there are objections, I'll go ahead and create them:

1. Top of the page adjustments: Change the nav menu and remove the social media links
2. Add About & FAQ pages (this ticket, no need for a new one)
3. Remove the text `Questions about speedupamerica.com? Find more information here.` from the home page (the only reason I'm suggesting a separate ticket for this one is to provide a separate place for discussion of it, it's a really easy and quick change)

Tagging @mattsayre and @alexandracorvello to get your thoughts.
Matt: Since we're adjusting/adding to the plan, should we make any adjustments to SPs for ticket 1 or 2 in my list above?

I'll make the new issues tomorrow afternoon if I haven't heard any objections by then.

**Comment by noahbrenner** [comment]
I've created new issues to split of some tasks from this one:

#249 for adding a nav menu - This one must be resolved first so that we *have* a nav menu to add About & FAQ links to, so I've added the `blocked` label to the current issue.

#250 for deleting the GitHub link from the home page - I'm interested in a little discussion on this one.

*This* issue can now focus just on creating the About and FAQ pages and adding them to the nav menu.

**Comment by alexandracorvello** [comment]
Thank you for doing that! Sounds like a good idea for putting the gihub link on the FAQ page.

Best,
Alexandra

**Comment by mattsayre** [comment]
The current version of the text for the FAQ page is in this directory https://drive.google.com/drive/folders/1IBkbK8LU7A4NYfiNKhgssaklLBjF0Z_S

**Comment by noahbrenner** [comment]
Hey @mattsayre and @alexandracorvello, I've got some questions for you in getting the About and FAQ pages up. Please let me know if you agree/disagree/don't have a strong opinion on what I have listed here.

## Questions/suggestions/info about the FAQ

* General note: I'm changing underlined text in the reference doc to bold text. On the web, it's bad UX to have underlined text that isn't a link.

Each of the headings below match a heading from the reference document that Matt linked to at the top of this issue. The bullet-points under them relate to that particular section.

### What types of broadband technology are there?

* How do you want to the footnote to look on the website? Instead of a footnote, I'd recommend a link displayed as "(source)", assuming there's a relevant URL for the evaluation. Another option is to show the footnote at the bottom of this section, which would be easier to find than putting at the end of the whole page.
* Is the "Hotspot wireless" paragraph meant to be its own definition section or is it a second paragraph of the "Wireless" definition?

### What are good internet speeds

* I corrected the math here: 1 Kbps = .001 Mbps (1/1,000), not .0001 (1/10,000).
* I don't think footnotes are as effective on the web as they are in print, but I see the value here, since the same footnote is referenced multiple times. I do think it would be better to provide a link, if possible, rather than just a text citation. Do you happen to have a link for footnote 3?

### What information do I need in order to take the test?

* I recommend against providing online instructions in PDF form. It's far more intuitive, more accessible, and less data-intensive to view a text-based webpage than to download a PDF. PDFs *are* useful if the purpose is to be able to download a copy or print it out. I suggest either removing the PDF link entirely or presenting it explicitly as a PDF that you can download/print.

### What is the difference between "my browser's location" and "use my address"?

* I suggest rewording
    - from: "have your internet browser ping your computer instead"
    - to: "have your internet browser provide your location instead"

### Do you collect my private data?

* I suggest changing the semicolon-delimited list to a bullet-point list.

### How can I tell if my connection is wired or wireless?

* I suggest adjusting wording to match the structure of the previous sentence:
    - from: "If you selected a wireless connection and there is no Ethernet cable, then your connection is wireless."
    - to: "If your computer connects to the internet without using a cable, then your connection is wireless"

### How can I view the source code?

* I suggest rewording the first sentence and changing what words are hyperlinked so that it's clearer that the link points to our repository, not to the GitHub homepage. I'd change it to:
    - You can view and/or download the source code from [SpeedUpAmerica's GitHub repository](https://github.com/Hack4Eugene/SpeedUpAmerica).

### How do I find specific results?

* Again, I recommend against using PDFs for online instructions. To get the page up quickly, I can link to these files, but I'd strongly prefer making them either distinct web pages or providing the instructions inline on this page (maybe collapsible, so they're not taking up visual space unless they're needed). What are your thoughts on this?
* The data downloaded is just a CSV file, so it's not specific to any spreadsheet program. I recommend changing this to something like:
    - "This will allow you to download the currently collected data and manipulate it in any spreadsheet program."
    - Or, if you really want to name specific programs: "This will allow you to download the currently collected data and manipulate it in any spreadsheet program, such as Microsoft Excel or LibreOffice Calc.

### How do I compare geographic regions?

* I recommend rewording this to clarify that the downloaded file is not Excel-specific:
    - "The best way is to download the data in CSV format by going to the bottom of [...]. You can import this file into any spreadsheet program in order to filter, sort, and manipulate the data [...]"
* I'll also make the text "Results page" a link.

**Comment by alexandracorvello** [comment]
Hello Noah,
Thank you so much for reviewing the FAQ page so thoroughly! I have put my comments on your different recommendations below in red. I agree with most of them.

Best,
Alexandra

**Comment by noahbrenner** [comment]
@alexandracorvello  Sounds great, thanks! I'll go with those answers.

I'll certainly link to the PDFs for now, but for future updates here's something to keep in mind about the accessibility of PDFs: A PDF has a rigid page size, whereas a webpage can easily adapt to the width of the screen. This means that if someone views a PDF on a mobile device, for example, the text will be too small to read, or they'll have to zoom in and scroll horizontally to read a single line of text. It's still plenty easy to print a webpage, so a PDF isn't required for that. PDFs can also be less consistent when trying to copy/paste text and different browsers have different defaults for accessing them (download, view in the browser, view in an external program, etc.).

There's a benefit in distinguishing the purpose of different media types and I'm in support of providing a PDF to download for those who want it, but I strongly suggest (in the long term) that it be a *supplemental* option. I'd argue that providing the info on a webpage is *more* user-friendly than only providing a PDF.

For now, going with the PDFs, here's one more question. Should we:

1. Host the PDF files on the SpeedUpAmerica website?
    - PRO: People may find an in-site link more trustworthy and professional and it would have a cleaner URL that's easier to write down.
    - PRO: No worrying about Google Drive read/write/access permissions.
    - CON?: If a file needs to be updated, you'll probably need one of us to update it on the site, rather than changing it in your own Google Drive.
2. Or link to the files on Google Drive?
    - PROs/CONs: the reverse of those for option 1.

I prefer option 1, but what do you think?

**Comment by alexandracorvello** [comment]
I like option 1 as well because right now the information fits with what’s on the website and if anything changes I’m hoping this could be updated as the change it made.

Best,
Alexandra
